src/test/cli/rbd/help.t

```diff
@@ -420,6 +423,55 @@
     -p [ --pool ] arg    pool name
     --group arg          group name
   
+  rbd help group image add
+  usage: rbd group image add [--pool <pool>] [--group <group>] 
+                             <group-spec> <image-spec> 
+  
+  Add an image to a consistency group.
+  
+  Positional arguments
+    <group-spec>         group specification
+                         (example: [<pool-name>/]<group-name>)
+    <image-spec>         image specification
+                         (example: [<pool-name>/]<image-name>)
+  
+  Optional arguments
+    -p [ --pool ] arg    pool name
+    --group arg          group name
```

Comment:
I think it makes sense. The CLI becomes a little bit unstructured, but at least it should be convenient to use. I'll use --group-pool and --image-pool, if any of them is missing or both of them then I'll take their values from --pool, if even that one is missing then I'll use the default pool name.